.github/workflows/golangci-lint.yaml

```diff
@@ -7,9 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+      - uses: actions/checkout@v3
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
```

Comment:
same ci fix is included in this PR too: https://github.com/cohere-ai/cohere-go/pull/41/
I'd personally suggest just opening (and immediately merging 😅) a small PR/direct-commit for this instead

Reply:
I think we'll just merge it with this, and then the diff will disappear from the other PR